lib/clients.sh

```diff
@@ -115,7 +116,8 @@ function build-deisctl {
 function setup-deisctl-client {
   local version="${1}"
 
-  if is-released-version "${version}"; then
+  if is-released-version "${version}" && \
+     ! is-rc-version "${version}"; then
```

Comment:
After talking to @mboersma - this will change back to just a check to see if it's a released version after the 1.11 release because we'll be building and pushing the clients up during release candidate testing.